openassessment/xblock/submission_mixin.py

```diff
@@ -505,7 +505,11 @@ def get_download_urls_from_submission(self, submission):
                 file_download_url = self._get_url_by_file_key(key)
                 if file_download_url:
                     file_description = descriptions[idx].strip() if idx < len(descriptions) else ''
-                    file_name = file_names[idx].strip() if idx < len(file_names) else ''
+                    try:
+                        file_name = file_names[idx].strip() if idx < len(file_names) else ''
+                    except AttributeError:
+                        file_name = ''
+                        logger.error('descriptions[idx] is None in {}'.format(submission))
```

Comment:
@AhtishamShahid are these temporary logs?

Reply:
Yes, it is a temporary log that will be removed after the root cause of the issue is found.